pkg/grpc/user_devices.proto

```diff
@@ -71,6 +71,12 @@ message UserDevice {
   string postal_code = 17;
   string geo_decoded_country = 18;
   string geo_decoded_state_prov = 19;
+  optional SyntheticDevice syntheticDevice = 20;
+}
+
+message SyntheticDevice {
+  int64 tokenId = 1;
```

Comment:
The protobuf style for fields is snake case, let's do that. `token_id`, `integration_token_id`. I was thinking of the JSON.

Reply:
Is any code populating these fields?